modules/javafx.controls/src/test/java/test/javafx/scene/chart/XYNumberLineChartsTest.java

```diff
@@ -25,11 +25,15 @@
 
 package test.javafx.scene.chart;
 
+import javafx.scene.shape.MoveTo;
+import javafx.scene.shape.Path;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Note: This mixes JUnit 4 with JUnit 5 API. AFAIK, this is nothing we want, so it is better to use the already imported `assertEquals` here.

Reply:
right, I've changed this to JUnit 4 and removed the JUnit 5 import.